apps/dashboard/src/components/primitives/code-block.tsx

```diff
@@ -69,7 +102,14 @@ interface CodeBlockProps {
  *   title="Configuration"
  * />
  */
-export function CodeBlock({ code, language = 'typescript', title, className, secretMask = [] }: CodeBlockProps) {
+export function CodeBlock({
+  code,
+  language = 'typescript',
+  theme = 'dark',
+  title,
+  className,
+  secretMask = [],
+}: CodeBlockProps) {
```

Comment:
I've updated the code-block component to be used across all the dashboard areas including workflow trigger page